packages/story-editor/src/components/floatingMenu/elements/borderWidthAndColor.js

```diff
@@ -135,8 +136,11 @@ function BorderWidthAndColor() {
               value={border.color || BLACK}
               onChange={handleColorChange}
               hasInputs={false}
-              hasEyeDropper={false}
               allowsOpacity={canHaveBorderOpacity}
+              hasEyedropper={false}
+              allowsGradient={false}
+              allowsSavedColors={false}
+              changedStyle={'border-color'}
```

Comment:
Don't need string interpolation here
```suggestion
              changedStyle="border-color"
```